replay_build_scripts/upload_build_artifacts.mjs

```diff
@@ -130,20 +131,20 @@ function prepareWindowsBinaries(buildId) {
   const buildArchive = `${buildId}.zip`;
   fs.rmSync("replay-chromium", { force: true, recursive: true });
   fs.mkdirSync("replay-chromium");
-  spawnChecked(
```

Comment:
this code was just totally hosed, I suspect it was a copy paste that had never actually ran and so we never fixed.